plugins/catalog-backend-module-unprocessed/src/UnprocessedEntitiesModule.ts

```diff
@@ -103,7 +112,31 @@ export class UnprocessedEntitiesModule {
     return res;
   }
 
-  registerRoutes() {
+  registerRoutes({ permissions }: { permissions: PermissionEvaluator }) {
+    const permissionIntegrationRouter = createPermissionIntegrationRouter({
+      permissions: [unprocessedEntitiesDeletePermission],
+    });
+
+    const isRequestAuthorized = async (
+      req: Request,
+      permission: BasicPermission,
+    ): Promise<boolean> => {
+      const token = getBearerTokenFromAuthorizationHeader(
```

Comment:
@camilaibs does this need converting to use the new auth APIs? Or do permissions calls just forward the token blindly?

Reply:
`.authorize` [now accepts `credentials` directly](https://github.com/backstage/backstage/blob/bf2c435983b18cc6674c226b82d2694557e53252/plugins/azure-sites-backend/src/service/router.ts#L115), instead of a token